src/config.ts

```diff
@@ -668,6 +678,21 @@ export const configure = async (customProjectConfig?: CustomProjectConfig) => {
     ...projectConfig,
   };
 
+  if (isLocalDebugMode()) {
+    config.generateOnly = true;
+    config.lostPixelProjectId = undefined;
+
+    if (
+      config.pageShots?.baseUrl.includes('http://') &&
+      !config.pageShots?.baseUrl.includes('localhost')
```

Comment:
good points, not sure if anybody uses https on actions, as for 127.0.01 - I guess that will just work locally as well, no need to change

Reply:
if someone specifies the baseUrl as `http://127.0.0.1:3000/...` in `lostpixel.config.ts` it would be omitted.
so a check for that should be there too.

Reply:
done ✅

Reply:
[f56a6de](https://github.com/lost-pixel/lost-pixel/pull/328/commits/f56a6deeb6e44c763226397e63bf94b9a8fae5eb)